sonar-project.properties

```diff
@@ -1,3 +1,7 @@
 sonar.projectKey=DataBiosphere_terra-ui
 sonar.organization=broad-databiosphere
-sonar.javascript.lcov.reportPaths=coverage/lcov.info
+
+# Only scan code under src.
+sonar.sources=src
+# Skip test files, which can be anywhere in the source tree.
+sonar.exclusions=**/**.test.js
```

Comment:
I've read that this is a default but it doesn't seem to be honored as the `test.js` files are currently showing up in the sonar coverage reports.